### ISSUE TITLE
initializationOptions: Cannot read property 'config' of undefined

### ISSUE THREAD

**Issue by last-partizan** [vuejs/vetur#1341]
Hi, i'm trying to use vue-language-server with vim and LanguageClient-neovim and it does not work.

I traced bug to this error:
```
00:54:01 WARN unnamed src/language_server_protocol.rs:2132 Failed to start language server automatically. Error: Failure { jsonrpc: Some(V2), error: Error { code: InternalError, message: "Request initialize failed with message: Cannot read property \'config\' of undefined", data: None }, id: Num(1) }
```
https://github.com/autozimu/LanguageClient-neovim/blob/next/src/language_server_protocol.rs#L897

Looks like initializationOptions can be null, but vue-language-server does not take this into account here:

https://github.com/vuejs/vetur/blob/82d16031f58e0f54fde1fce94cf2c9518fe5080e/server/src/services/vls.ts#L83

It looks like trivial issue, can you fix it or should i make pull-request?

**Comment by jonboiser** [comment]
In my (unsuccessful) attempts to use VLS in atom, I have gotten rid of the errors by moving `config` into the path array.

It clears up the errors, but I am still not getting messages from the language server in atom.

```diff
+  logger.setLevel(_.get(params.initializationOptions.config, ['vetur', 'dev', 'logLevel'], 'INFO')); 
-  logger.setLevel(_.get(params.initializationOptions, ['config', 'vetur', 'dev', 'logLevel'], 'INFO')); 

```

**Comment by zulus** [comment]
Any workaround?

**Comment by octref** [comment]
Isn't this already fixed?
https://github.com/vuejs/vetur/blob/master/server/src/config.ts

**Comment by zulus** [comment]
No, without my patch I still see same error.

**Comment by octref** [comment]
Fixed by #1705.
